ledger-tool/src/main.rs

```diff
@@ -1933,6 +1933,7 @@ fn main() {
                         Some(snapshot_version),
                         output_directory,
                         ArchiveFormat::TarZstd,
+                        &bank.get_thread_pool(),
```

Comment:
This could use threadpool with all threads.

Reply:
will address in followup pr